Tasks/app-store-release/app-store-release.ts

```diff
@@ -33,7 +33,7 @@ function isValidFilePath(filePath: string): boolean {
 // Attempts to find a single ipa file to use by the task.
 // If a glob pattern is provided, only a single ipa is allowed.
 function findIpa(ipaPath: string): string {
-    let paths: string[] = tl.glob(ipaPath);
+    let paths: string[] = tl.findMatch('', ipaPath);
```

Comment:
Is it working the same as for previous lib?

Reply:
The previous lib was [using](https://github.com/microsoft/azure-pipelines-task-lib/blob/releases/0.8/node/lib/task.ts#L867) npm package [glob](https://www.npmjs.com/package/glob). Azure-pipelines-task lib doesn't have `glob` property but [has ](https://github.com/microsoft/azure-pipelines-task-lib/blob/master/node/task.ts#L1501) `findMatch `property which does the same in this case

Reply:
[Replaced](https://github.com/microsoft/app-store-vsts-extension/pull/170/commits/16c3242ae7958f612e1545a660505506aa1caa2e) glob answers by findMatch in tests. All unit-tests passed

![image](https://user-images.githubusercontent.com/64020607/94072299-15f32c00-fdfe-11ea-8c90-2c04048f81a6.png)